src/bynder-js-sdk.js

```diff
@@ -529,6 +529,90 @@ export default class Bynder {
         return request.send();
     }
 
+    /**
+     * Add an option of metaproperty
+     * @see {@link http://docs.bynder.apiary.io/#reference/metaproperties/specific-metaproperty-operations/create-metaproperty-option|API Call}
+     * @param {Object} queryObject={} - An object containing the id of the desired metaproperty.
+     * @param {String} queryObject.id - The id of the desired metaproperty.
+     * @param {String} queryObject.name - The name of the desired metaproperty.
+     * @return {Promise} Metaproperty - Returns a Promise that, when fulfilled, will either return an Object with the
```

Comment:
The `Response` thing again

Reply:
I guess this applies to all these new methods.

Reply:
Good finding. Will change that